bnelson777/id.ly
# IL-75: Add share and new message button to card view

-Added new buttons to card view

Testing instructions:
1. Choose card from wallet and from rolodex.
2. For each card verify buttons appear in card view.
3. Verify Share button leads to Share screen and Message button leads to new message screen.
